web/landing_page.go

```diff
@@ -71,7 +72,7 @@ var (
 	landingPagecssContent string
 )
 
-func NewLandingPage(c LandingConfig) (*LandingPageHandler, error) {
+func NewLandingPage(c LandingConfig, routePrefixed bool) (*LandingPageHandler, error) {
```

Comment:
We already have a `LandingConfig`, it doesn't make sense to add another `bool` value here.

Also, rather than have a bool value, we can add a `RoutePrefix string` directly to the `LandingConfig`. If the `RoutePrefix` is `""`, we can default it to `/`.